jbi/models.py

```diff
@@ -359,7 +360,25 @@ class ActionContext(Context, extra=Extra.forbid):
 
     rid: str
     operation: Operation
+    current_step: Optional[str]
     event: BugzillaWebhookEvent
     jira: JiraContext
     bug: BugzillaBug
     extra: dict[str, str] = {}
+    responses_by_step: dict[str, list] = {}
```

Comment:
I think this will work too?
```suggestion
    responses_by_step: DefaultDict[str, list] = defaultdict(list)
```

Reply:
That was my first attempt, I can't remember why I changed it...